libs/wingc/src/docs.rs

```diff
@@ -217,7 +217,11 @@ fn render_signature_help(f: &FunctionSignature) -> String {
 		let param_type_unwrapped = param.typeref.maybe_unwrap_option();
 		let is_last = param_idx == f.parameters.len() - 1;
 
-		let param_name: &String = &param.name;
+		let param_name = if param.name.is_empty() {
+			param_idx.to_string()
+		} else {
+			param.name.clone()
+		};
```

Comment:
I'm curious where this situation arises?

Reply:
function types as annotations

e.g. hover over `x` [here](https://www.winglang.io/play/?code=bABlAHQAIAB4ADoAIAAoAG4AdQBtACkAOgAgAHYAbwBpAGQAIAA9ACAAKABhADoAIABuAHUAbQApACAAPQA%2BACAAewB9ADsA)

Reply:
Got it. Maybe name them `arg0`, `arg1`, etc. by default?

Reply:
I'm good with that. I didn't do it initially cause I was worried it would be misleading (since they don't have names) but it's probably better.